Parsers/ForgeRockParser.txt

```diff
@@ -14,5 +14,5 @@
 //
 
 CommonSecurityLog
-| where DeviceVendor += ("ForgeRock Inc")
+| where DeviceVendor ~= ("ForgeRock Inc")
```

Comment:
This should be =~.  Please change the order and also remove the parentheses.

`| where DeviceVendor =~ "ForgeRock Inc"`